src/features/instance/operations/mutations/updateComponentFile.ts

```diff
@@ -0,0 +1,28 @@
+import { useMutation } from '@tanstack/react-query';
+import { instanceClient } from '@/config/instanceClient';
+
+type SetComponentFileRequest = {
+	file: string;
+	payload?: string;
+	project: string;
+};
```

Comment:
I dream of a day when we have OpenAPI types for these instance operations too 😆

Reply:
😂 😂